development/receptor_configs/node01.yml

```diff
@@ -0,0 +1,30 @@
+---
+- node:
+    id: controller01
+
+- log-level:
+    level: debug
+
+- tcp-peer:
+    address: node02.example.internal:7323
+
+- control-service:
+    service: control
+    filename: /tmp/receptor.sock
+    tcplisten: "0.0.0.0:1234"
+
+- work-command:
+    worktype: gather-hostname
+    command: cat
+    params: /etc/hostname
+
+- work-command:
+    worktype: hello-world
+    command: python3
+    params: /etc/receptor/examples/example_python.py
+
+- work-command:
+    worktype: run-nautobot-job
+    command: nautobot-server
+    params: runjob
+    allowruntimeparams: true
```

Comment:
This command runs `nautobot-server runjob`. ` -u admin example_app.jobs.ExampleJob` is passed via the --param flag.